src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -60,16 +82,28 @@
     <!-- Append the xunit trait options to the end of the TestCommandLine -->
     <PropertyGroup>
       <TestCommandLine>$(TestCommandLine.Replace('{XunitTraitOptions}', '$(XunitTraitOptions)'))</TestCommandLine>
+      <EventTracerDataFile>$(TargetFileName)%(TestTargetFrameworkBatch.Count).etl</EventTracerDataFile>
+      <EventTracerMetrics>-c $(ClrMetrics) -k $(KernelMetrics)</EventTracerMetrics>
+      <EventTracerReport>Perf.$(MSBuildProjectName)%(TestTargetFrameworkBatch.Count).xml</EventTracerReport>
```

Comment:
Can we try to keep most of these in the perf testing targets? It looks like you are putting it here for the batching parameters but in general I think the directory structure will be able to keep these files from clashing.